api/RTCRtpReceiver.json

```diff
@@ -11,7 +11,7 @@
             "version_added": "59"
           },
           "edge": {
-            "version_added": null
+            "version_added": true
```

Comment:
This and `getStats` confirmed by https://software.hixie.ch/utilities/js/live-dom-viewer/saved/6830 in Edge 18.